tests/Unit/Core/Foundation/VersionTest.php

```diff
@@ -509,8 +509,8 @@ public function provideVersions()
                     'buildMeta' => 'nightly.20190526',
                 ],
             ],
-            '8.1.2' => [
-                '8.1.2',
+            '1.8.1.2' => [
+                '1.8.1.2',
```

Comment:
```suggestion
            '8.1.2' => [
                '8.1.2',
```

It should be that because the version passed is 1.8.1.2 like above with `1.7.7.0`